CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # 2.63.x
+* Update Roslyn to ...
```

Comment:
>... [](http://example.com/codeflow?start=19&length=3)

Did you mean to add a value here?

Reply:
Yeah, I just don't know what it is yet :)

Reply:
Now updated :)